src/Features/RemotePaymentMethods/DefaultPaymentGateways.php

```diff
@@ -70,6 +70,26 @@ public static function get_all() {
 					),
 				),
 			),
+			array(
+				'key'        => 'ppcp-gateway',
+				'title'      => __( 'PayPal Payments', 'woocommerce-admin' ),
+				'content'    => __( "Safe and secure payments using credit cards or your customer's PayPal account.", 'woocommerce-admin' ),
+				'image'      => WC()->plugin_url() . '/assets/images/paypal.png',
+				'plugins'    => array( 'woocommerce-paypal-payments' ),
+				'is_visible' => array(
+					(object) array(
+						'type'      => 'base_location_country',
+						'value'     => 'IN',
+						'operation' => '!=',
+					),
+					(object) array(
+						'type'        => 'option',
+						'option_name' => 'woocommerce_onboarding_profile',
+						'value'       => 'cbd-other-hemp-derived-products',
+						'operation'   => '!contains',
```

Comment:
Same comment [here](https://github.com/woocommerce/woocommerce-admin/pull/7025#discussion_r638361068) about this CDB check being faulty.

Reply:
Thanks!  Added the same CBD logic used in https://github.com/woocommerce/woocommerce-admin/pull/7025